ci-operator/config/openshift/installer/openshift-installer-master__altinfra.yaml

```diff
@@ -72,26 +58,10 @@ resources:
     requests:
       cpu: 100m
       memory: 200Mi
-  govet:
-    requests:
-      cpu: "3"
-      memory: 5Gi
   installer:
     requests:
       cpu: "3"
       memory: 5Gi
-  installer-artifacts:
-    requests:
-      cpu: "3"
-      memory: 5Gi
-  unit:
-    requests:
-      cpu: "3"
-      memory: 5Gi
-  upi-installer:
-    requests:
-      cpu: "3"
-      memory: 5Gi
 test_binary_build_commands: |
   # Some agent-based installer tests require nmstatectl
   dnf install -y nmstate
```

Comment:
Without any agent related tests then this could be skipped

Reply:
thanks
done